src/Item_Disk.php

```diff
@@ -232,100 +225,90 @@ public static function showForItem(CommonDBTM $item, $withtemplate = 0)
-            $header .= "<th>" . __('Free percentage') . "</th>";
-            $header .= "<th>" . __('Encryption') . "</th>";
-            $header .= "</tr>";
-            echo $header;
-
-            Session::initNavigateListItems(
-                __CLASS__,
-                //TRANS : %1$s is the itemtype name,
-                           //        %2$s is the name of the item (used for headings of a list)
-                                          sprintf(
-                                              __('%1$s = %2$s'),
-                                              $item::getTypeName(1),
-                                              $item->getName()
-                                          )
-            );
 
-            $disk = new self();
-            foreach ($iterator as $data) {
-                $disk->getFromResultSet($data);
-                echo "<tr class='tab_bg_2" . (isset($data['is_deleted']) && $data['is_deleted'] ? " tab_bg_2_2'" : "'") . "'>";
-                echo "<td>" . $disk->getLink() . "</td>";
-                echo "<td>" . Dropdown::getYesNo($data['is_dynamic']) . "</td>";
-                echo "<td>" . $data['device'] . "</td>";
-                echo "<td>" . $data['mountpoint'] . "</td>";
-                echo "<td>" . $data['fsname'] . "</td>";
-                //TRANS: %s is a size
-                $tmp = Toolbox::getSize($data['totalsize'] * 1024 * 1024);
-                echo "<td>$tmp</td>";
-                $tmp = Toolbox::getSize($data['freesize'] * 1024 * 1024);
-                echo "<td>$tmp</td>";
-                echo "<td>";
-                $percent = 0;
-                if ($data['totalsize'] > 0) {
-                    $percent = round(100 * $data['freesize'] / $data['totalsize']);
-                }
-                $rand = mt_rand();
-                Html::progressBar("percent$rand", [
-                    'create'  => true,
-                    'percent' => $percent,
-                    'message' => "$percent %",
+        $disk = new self();
+        $entries = [];
+        foreach ($iterator as $data) {
+            $disk->getFromResultSet($data);
+            $used = $data['totalsize'] - $data['freesize'];
+            $usedpercent = 0;
+            if ($data['totalsize'] > 0) {
+                $usedpercent = round(100 * $used / $data['totalsize']);
+            }
+
+            $encryption_label = '';
+            if ($data['encryption_status'] !== self::ENCRYPTION_STATUS_NO) {
+                $twig_params = [
+                    'encryption_status' => Dropdown::getYesNo($data['encryption_status'] === self::ENCRYPTION_STATUS_YES),
+                    'encryption_tool' => $data['encryption_tool'],
+                    'encryption_algorithm' => $data['encryption_algorithm'],
+                    'encryption_type' => $data['encryption_type'],
+                ];
+                $encryptionTooltip = TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
```

Comment:
@cconard96 This have to be changed. Translations here will not be detected by `xgettext`.

I suggets to move the whole HTML rendering of the current method into a dedicated template.